spec/services/filesync/directory.spec.ts

```diff
@@ -317,8 +317,6 @@ describe("Directory.hashes", () => {
     } else {
       expect(mapValues(hashes, (hash) => hash.permissions)).toEqual(mapValues(hashes, () => undefined));
     }
-
-    expect.assertions(2);
```

Comment:
I had to remove this because the new `mock` function, which is used a few times before all tests, uses `expect` on its arguments making this assertion fail.